ChannelManagementV4/Management_Channel_API_v4.postman_collection.json

```diff
@@ -0,0 +1,1707 @@
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n    \"data\": {\n        \"attributes\": {\n            \"name\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ],\n            \"description\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ],\n            \"images\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ],\n            \"channel_number\": {\n                \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n            },\n            \"media_type\": \"<string>\",\n            \"enabled_services\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ],\n            \"restrictions\": {\n                \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n            },\n            \"visibility\": {\n                \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n            },\n            \"dvb_info\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ],\n            \"metadata_provider_information\": [\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                },\n                {\n                    \"value\": \"<Error: Too many levels of nesting to fake this schema>\"\n                }\n            ]\n        },\n        \"id\": \"<string>\",\n        \"type\": \"<string>\"\n    }\n}"
+						},
+						"url": {
+							"raw": "{{baseUrl}}/api/channel/management/v4/:channel_id",
+							"host": [
+								"{{baseUrl}}"
+							],
+							"path": [
+								"api",
+								"channel",
+								"management",
+								"v4",
+								":channel_id"
+							],
+							"variable": [
+								{
+									"key": "channel_id",
+									"value": "<number>",
```

Comment:
Tiny thing, just curious why only "Update channel" has `<number>` as default value in Postman, 
![image](https://user-images.githubusercontent.com/29536745/116020360-b5202080-a678-11eb-9153-ccf17fac5102.png)
while "Get channel" and "Delete channel" are empty.
![image](https://user-images.githubusercontent.com/29536745/116020395-c36e3c80-a678-11eb-953e-48c024d89c0d.png)

Reply:
Hm, I am not sure, seem like postman parses it from API in that way, at least do not remember I was changing it.